src/webnn_native/openvino/GraphIE.cpp

```diff
@@ -63,16 +63,56 @@ namespace webnn_native { namespace ie {
             return ieDesc;
         }
 
-        ie_conv2d_options Conv2dOptionsForIE(Conv2dOptions const* options) {
-            ie_conv2d_options ieOptions;
-            ieOptions.padding = options->padding;
-            ieOptions.strides = options->strides;
-            ieOptions.dilations = options->dilations;
-            ieOptions.groups = options->groups;
-            ieOptions.inputLayout = static_cast<ie_input_operand_layout>(options->inputLayout);
-            ieOptions.filterLayout = static_cast<ie_filter_operand_layout>(options->filterLayout);
-            ieOptions.autoPad = static_cast<ie_auto_pad>(options->autoPad);
-            return ieOptions;
+        ie_activation createActivationForIE(OperatorBase* activation) {
+            ie_activation ieActivation;
+            if (activation == nullptr) {
+                return ieActivation;
+            }
+            auto operatorType = activation->GetOperatorType();
+            switch (operatorType) {
+                case OperatorType::None:
+                    break;
+                case OperatorType::Clamp: {
+                    ieActivation.type = ie_activation_type::Clamp;
+                    auto clamp = reinterpret_cast<op::ClampOperator*>(activation);
+                    auto options = clamp->GetOptions();
+                    if (options->minValue != nullptr) {
+                        op::Constant* minConstant =
+                            reinterpret_cast<op::Constant*>(options->minValue);
```

Comment:
Should you check whether the the `minValue` is a constant?

Reply:
It is not easy to check whether a node is a constant here, oprand in operator options will not call [AddConstant](https://github.com/webmachinelearning/webnn-native/blob/main/src/webnn_native/openvino/GraphIE.cpp#L142) function in implementation. And it is a common problem, I will open an issue for discussion.

Reply:
> oprand in operator options will not call [AddConstant](https://github.com/webmachinelearning/webnn-native/blob/main/src/webnn_native/openvino/GraphIE.cpp#L142) function in implementation.

I suppose every constant should be created by `AddConstant`, am I wrong? So for a backend implementation, it would be possible to record every constant operands in a set. And when handling operands in options, it could check the operand against the constant set, then it could be possible to know whether an operand is a constant or not. This is one possible way in my mind. And we can also add support in the infrastructure to mark an operand is a constant or not. @fujunwei .

Reply:
Yes, we must make sure the `OperandBase` in the `mOperandIdMap` not only `Constant `Operand but also intermediate Operand.
 
For Constant Operand,  there are two approach to make sure it's constant Operand, one is simple with adding a flag in `OperandBase`, but intermediate Operand must be in the graph. 

Another is continue using current design,  and return the fused OperandBase, for example fused Clamp,
https://github.com/webmachinelearning/webnn-native/blob/main/src/webnn_native/GraphBuilder.cpp#L92
OperandBase* GraphBuilderBase::Conv2d(OperandBase* input,
                                          OperandBase* filter,
                                          Conv2dOptions const* options) {
        DAWN_VALIDATE_AND_INFER_TYPES(new op::Conv2d(this, input, filter, options));
    }

=>

OperandBase* GraphBuilderBase::Conv2d(OperandBase* input,
                                          OperandBase* filter,
                                          Conv2dOptions const* options) {
        OperandBase*  conv2d = new op::Conv2d(this, input, filter, conv2dOptions);
        DAWN_VALIDATE_AND_INFER_TYPES(new op::Clamp(this, conv2d , clampOptions));
    }

Reply:
Thanks, will update as this.